brave/brave-ios
# Fix #3396: Add ability to search Open Tabs

This pull request contains small alterations in Tab Tray UX and ability to search Open Tabs inside TabTray without changing other Tab Tray abilities.

## Summary of Changes


This pull request fixes #3396

## Submitter Checklist:


## Test Plan:

- Open tab tray 
- Try searching different queries
- Switch to Private mode and check the bar is not presented when private mode information is presented
- Add some tabs in private mode
- Test Private mode is working as intended for search


## Screenshots:

https://user-images.githubusercontent.com/6643505/143916721-36695345-d8e9-4b43-b81d-cbae420bbfbf.MP4

A screenshot from iPhone 8 with light mode

![tasearchsmallosmall](https://user-images.githubusercontent.com/6643505/149013252-07319a6a-22c7-48d7-9a2d-e7ced1853898.png)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
